app/legacy-processing/calculate-approximate-re-received-datetime.js

```diff
@@ -1,8 +1,8 @@
 const calculateApproximateREReceivedDateTime = (primaryPaymentRequest, paymentRequest) => {
-  if (primaryPaymentRequest.debtType && paymentRequest.completedPaymentRequests[0]) {
+  if (primaryPaymentRequest.debtType && paymentRequest.completedPaymentRequests?.[0]) {
```

Comment:
that has taught me something new today. I had to lookup that syntax as I have not seen it before.